addons/l10n_in/tests/test_partner_details_on_invoice.py

```diff
@@ -0,0 +1,141 @@
+from odoo.addons.account_reports.tests.common import TestAccountReportsCommon
```

Comment:
You are trying to import from enterprise in community....

Reply:
@vlst-odoo  Indeed...

@hamo-odoo We might either move this to the l10n_in_reports module or use the regular InvoiceCommon test

Reply:
WTF! How did I end up making this mistake 🤦🏻‍♂️?! The test case is for the community we need to change inherit class I guess